packages/gatsby-theme-docs/src/components/link.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Location } from '@reach/router';
+import { Link as GatsbyLink, withPrefix } from 'gatsby';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import { OutboundLink } from 'gatsby-plugin-google-analytics';
+import ExternalLinkIcon from '../icons/external-link-icon.svg';
+import { colors, dimensions } from '../design-system';
+import { useSiteData } from '../hooks/use-site-data';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+const productionHostname = 'docs.commercetools.com';
```

Comment:
@nkuehn should we make this configurable? Or do you want to keep it hardcoded?

Reply:
That's a good general question - we might need the official domain e.g. for the canonical links, too and we likely also need it for RSS feeds later on.  

We could force setting it in the site metadata in the config and have a default in the theme. I would not hardwire it inside the link implementation though.   

BTW that's a typical case where you may need a static query in a component and think about testability.

Reply:
Sounds good to me to put it in the metadata